components/Table.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react"
+import { 
+  useTable,
+  Column,
+} from "react-table"
+import { 
+  Table as ChakraTable,
+  Thead,
+  Tbody,
+  Tr,
+  Th,
+  Td,
+} from "@chakra-ui/react"
+
+export interface TableProps {
+  columns: Column<object>[]
+  data: object[]
+}
+
+export const Table = ({columns, data}: TableProps ): JSX.Element =>  {
+  const {
+    getTableProps,
+    getTableBodyProps,
+    headerGroups,
+    rows,
+    prepareRow,
+  } = useTable({
```

Comment:
This code is code is basically copied from [Chakra's docs on integrating React Table](https://chakra-ui.com/guides/integrations/with-react-table).